docs/getting_started.md

```diff
@@ -22,103 +35,118 @@ IDE extension.
 
 ## Configuration
 
-Once the VSCode extension is installed, you are presented with the "Set up
-Konveyor" page. This page is where you as a user can configure Kai to suit your
-needs. This page can also be accessed by clicking "Help"->"Welcome" and finding
-"Set up Konveyor" in the "Walkthroughs" section.
+<iframe width="560" height="315" src="https://www.youtube.com/embed/IBa8B_RluGs" title="Kai configuration walkthrough" frameborder="0" allowfullscreen></iframe>
```

Comment:
@Pkylas007 I don't see this HTML tag rendering as you intended.

This is what I see with a preview:
<img width="1193" height="166" alt="Screenshot 2025-08-13 at 10 44 50 AM" src="https://github.com/user-attachments/assets/6f31f94e-9f63-415f-8400-cecee767e772" />